README.md

```diff
@@ -144,29 +144,78 @@ To apply the __buttontabs__ styling, add a `o-tabs--buttontabs` class to the roo
 <ul data-o-component="o-tabs" class="o-tabs o-tabs--buttontabs" role="tablist">
 ```
 
-or use the mixin:
+The __buttontabs__ style comes in two sizes that conform to the `o-buttons` sizes: medium and big. Medium is the default size and big can be applied by adding the class `o-tabs--big`.
+
+#### Options
+
+* __Align right__: Add `o-tabs--alignright` to the root element.
+
+#### Silent mode
+
+As with all Origami components, o-tabs has a [silent mode](https://origami.ft.com/spec/v1/sass/#sass-silent-mode). To use its compiled CSS (rather than incorporating its mixins into your own Sass) set `$o-tabs-is-silent: false;` in your Sass before you import the o-tabs Sass:
 
 ```sass
-@include oTabsButtonTabs;
+$o-tabs-is-silent: false;
+@import 'o-tabs/main';
 ```
 
-The __buttontabs__ style comes in two sizes that conform to the `o-buttons` sizes: medium and big. Medium is the default size and big can be applied by adding the class `o-tabs--big`.
+#### Mixin: `oTabs`
 
-#### Options
+If using o-tabs in silent mode, you'll need to use the mixins outlined here to output styles.
 
-* __Align right__: Add `o-tabs--alignright` to the root element.
+The `oTabs` mixin is used to output base styles as well as styles for _all_ of the tab themes. This output includes the `o-tabs` classes:
+
+```scss
+@include oTabs();
+```
 
-#### Mixins
+```css
+.o-tabs {
+    /* styles */
+}
+.o-tabs--buttontabs {
+    /* styles */
+}
+.o-tabs--primary {
+    /* styles */
+}
+/* etc. */
+```
+
+If you wish to specify a subset of themes to output styles for, you can pass in options (see [themes](#themes) for available options):
 
-o-tabs styles are also available via mixins. The base styles and buttontabs styles are available using the `oTabs` and `oTabsButtonTabs` mixins.
+```scss
+@include oTabs($opts: (
+    'themes': ('primary')
+));
+```
 
-If you're using the Sass mixins, you can also theme o-tabs using the `oTabsButtonTabsTheme` mixin, which can be set to any of the [`o-buttons` themes](https://github.com/Financial-Times/o-buttons#quick-start) by passing the name as a parameter:
+#### Mixin: `oTabsButtonTabsAddTheme`
```

Comment:
Does `oTabsButtonTabsAddTheme` need to be public if I can:
```
@include oTabs($opts: (
    'themes': ('primary', 'mono')
));
```

Reply:
Yep because `mono` isn't included by default, and the registry uses mono

Reply:
But the above snippet would work? As a Sass user I can include non-default themes without `oTabsButtonTabsAddTheme`? Should `oComponentAdd` be saved for custom things?

Reply:
Wait I have a suggestion, easier to commit it and I can roll back if necessary

Reply:
See https://github.com/Financial-Times/o-tabs/pull/80/commits/4023e81efc2f5dfad1ed7c0d6cd0b31a53c90286